canonicalwebteam/discourse/parsers/base_parser.py

```diff
@@ -243,111 +243,6 @@ def _parse_navigation(self, index_soup):
 
         return nav_html
 
-    def _parse_metadata(self, index_soup, section_name):
```

Comment:
Sorry to mess you around, but I just had a thought that maybe someone was actually using this parse_metadata function outside of this module. I did a search and found https://github.com/canonical/maas.io/blob/18ef395ef35b6cb3846ec029417d5ebfe8da0a19/webapp/doc_parser.py#L58 - maas is using it. I don't fully understand whether or how it's actually being used.

But I think it's probably safest for you to leave this function as is, and then create yourself a new `_parse_docs_metadata`, which is probably completely independent of this function. What do you think?

Reply:
sure thing, just re-added the original parse_metadata function